modules/hal_i2c/src/i2c_configs.hpp

```diff
@@ -0,0 +1,71 @@
+/* *******************************************************************************************
```

Comment:
Why another config file? there are no settings in here! Move it back to the i2c driver header plz